### PR TITLE
Webtestacceptance

### PR DESCRIPTION
**Pull request by acesiddhu**

### CODE REVIEW

File: test/TestAssets/QualityToolsAssets/Microsoft.TestPlatform.QTools.Assets.nuspec [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <description>MSDia typelib for reading debug symbols from managed code.</description>
+    <copyright>Copyright © Microsoft Corporation</copyright>
+    <contentFiles>
+      <!-- Include Assets as Content -->
+      <files include="**/*.*" buildAction="None" copyToOutput="true" flatten="false" />
+    </contentFiles>
+  </metadata>
+  <files>
+    <file src="WebTestProject\bin\*.*" target="contentFiles\any\any\WebTestAssets" />
```
Comment (smadala) [inline test/TestAssets/QualityToolsAssets/Microsoft.TestPlatform.QTools.Assets.nuspec]: `/s/WebTestProject/BingWebTest`.

**Review by smadala — APPROVED** [review]

File: test/Microsoft.TestPlatform.AcceptanceTests/DifferentTestFrameworkSimpleTests.cs [hunk 2]
```diff
@@ -113,5 +129,55 @@ private void CppRunAllTests(string runnerFramework, string platform)
...
+            var assemblyAbsolutePath = Path.Combine(this.testEnvironment.PackageDirectory, assemblyRelativePath);
+            var arguments = PrepareArguments(
+                assemblyAbsolutePath,
+                string.Empty,
+                string.Empty,
+                this.FrameworkArgValue);
+
+            arguments = string.Concat(arguments, $" /platform:{platform}");
+            this.InvokeVsTest(arguments);
+            this.StdOutputContains("WebTest1Coded");
```
Comment (Faizan2304) [inline test/Microsoft.TestPlatform.AcceptanceTests/DifferentTestFrameworkSimpleTests.cs]: We should check for summary here as well. 
`this.ValidateSummaryStatus(1, 0, 0);`

File: test/TestAssets/QualityToolsAssets/Microsoft.TestPlatform.QTools.Assets.nuspec [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (Faizan2304) [inline test/TestAssets/QualityToolsAssets/Microsoft.TestPlatform.QTools.Assets.nuspec]: Is there any specific reason for creating nuget package. Cant we use the Webtest directly as we are using other test assets like `SampleTestProject.csproj`

File: test/Microsoft.TestPlatform.AcceptanceTests/DifferentTestFrameworkSimpleTests.cs [hunk 4]
```diff
@@ -43,6 +43,22 @@ public void CPPRunAllTestExecutionPlatformx64(RunnnerInfo runnerInfo)
             AcceptanceTestBase.SetTestEnvironment(this.testEnvironment, runnerInfo);
             CppRunAllTests(runnerInfo.RunnerFramework, "x64");
         }
+		
+        [CustomDataTestMethod]
+        [NETFullTargetFramework]
+        public void WebTestRunAllTests(RunnnerInfo runnerInfo)
+        {
+            AcceptanceTestBase.SetTestEnvironment(this.testEnvironment, runnerInfo);
+            WebTestRunAllTests(runnerInfo.RunnerFramework, "x86");
```
[the remark this answers is not in the document]
Comment (acesiddhu) [reply test/Microsoft.TestPlatform.AcceptanceTests/DifferentTestFrameworkSimpleTests.cs]: fixed

File: test/Microsoft.TestPlatform.AcceptanceTests/DifferentTestFrameworkSimpleTests.cs [hunk 5]
```diff
@@ -113,5 +129,55 @@ private void CppRunAllTests(string runnerFramework, string platform)
...
+            var assemblyAbsolutePath = Path.Combine(this.testEnvironment.PackageDirectory, assemblyRelativePath);
+            var arguments = PrepareArguments(
+                assemblyAbsolutePath,
+                string.Empty,
+                string.Empty,
+                this.FrameworkArgValue);
+
+            arguments = string.Concat(arguments, $" /platform:{platform}");
+            this.InvokeVsTest(arguments);
+            this.StdOutputContains("WebTest1");
```
[the remark this answers is not in the document]
Comment (acesiddhu) [reply test/Microsoft.TestPlatform.AcceptanceTests/DifferentTestFrameworkSimpleTests.cs]: locally I have this.ValidateSummaryStatus(1, 0, 0);
but I guess the test might fail on other machines. I need to make changes to web test. let me do that and add this line in the test code

File: test/Microsoft.TestPlatform.AcceptanceTests/DifferentTestFrameworkSimpleTests.cs (hunk 2 above)
Comment (acesiddhu) [reply test/Microsoft.TestPlatform.AcceptanceTests/DifferentTestFrameworkSimpleTests.cs]: I guess this is same comment as above? will fix that

File: test/TestAssets/QualityToolsAssets/Microsoft.TestPlatform.QTools.Assets.nuspec (hunk 3 above)
Comment (acesiddhu) [reply test/TestAssets/QualityToolsAssets/Microsoft.TestPlatform.QTools.Assets.nuspec]: webtest is there in enterprise edition. you cant build in ur repo

**Pull request merged**